### PR TITLE
Fix continuous integration tests

### PR DESCRIPTION
**Pull request by barentsen**
In #144 I encountered several failures in the CircleCI continuous integration tests, which appear to be predominantly due to trivial typos in the `requirements.txt` files of a few notebooks.

This PR proposes to resolve those CI failures.

### CODE REVIEW

File: notebooks/MAST/TESS/interm_tess_prf_retrieve/requirements.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
 astropy==2.0.8
-astroquery==0.3.10.dev0
+astroquery==0.3.10
```
Comment (pllim) [inline notebooks/MAST/TESS/interm_tess_prf_retrieve/requirements.txt:2]: It is not wise to pin astroquery. Should always use the latest pre-release to pick up fixes for upstream service changes, but that is another story.

**Review by pllim — COMMENTED** [review]
Did you mean to rename a Tesscut notebook? Looks good otherwise. Thanks!

**Comment by barentsen** [comment]
> Did you mean to rename a Tesscut notebook?

I renamed
`MAST/TESS/interm_tesscut_dss_overlay/tesscut_dss_overlay.ipynb`
into
`MAST/TESS/interm_tesscut_dss_overlay/interm_tesscut_dss_overlay.ipynb`
because CircleCI (over in #144) appeared to fail the notebook with the following message
```
[NbConvertApp] WARNING | pattern 'interm_tesscut_dss_overlay' matched no files
```,
suggesting that the directory name and the notebook file names must match?

Indeed, after renaming, CircleCI in this PR no longer appears to fail the notebook.

Incidentally, this specific notebook does not currently appear in the index listing at https://spacetelescope.github.io/notebooks/ .  I wonder if this is caused by the name mismatch?

Note: having fixed the `requirements.txt` files for a few notebooks, I see that CircleCI now fails some of these notebooks with the same ` WARNING | pattern 'xxx' matched no files` message (i.e. fixing one issue revealed another).  Shall I go ahead and fix them as well?

**Comment by pllim** [comment]
Sure, if you want to fix them, I'll wait to merge. Thanks for the clarifications!

**Comment by pllim** [comment]
Hmm CircleCI didn't kick off https://app.circleci.com/pipelines/github/spacetelescope/notebooks?branch=pull%2F145

**Comment by barentsen** [comment]
Right. I only really created the second commit (fabfbf3) to see if I could trigger CircleCI. 🤔

**Comment by pllim** [comment]
@jbcurtin , do you know why CircleCI suddenly claims there is no workflow found here?

**Comment by barentsen** [comment]
@pllim I am happy for this PR to be merged!

There is one test failure left, in `notebooks/DrizzlePac/Initialization`, which appears to be a failure to install the pinned version of the `crds` package that is in that notebook's requirements file.  It is likely that the failure may be fixed by allowing a more recent version of `crds` to be used, but I did not attempt this fix as it may alter the contents of the notebook cells.

```
Initialization.tar Test - Notebooks Test Suite

/opt/conda/envs/notebooks_env/lib/python3.7/site-packages/virtualenv.py:1041: DeprecationWarning: the imp module is deprecated in favour of importlib; see the module's documentation for alternative uses
  import imp
    ERROR: Command errored out with exit status 1:
     command: /tmp/Initializationyciqwo9_/env/bin/python3 -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-bhgwge4w/crds/setup.py'"'"'; __file__='"'"'/tmp/pip-install-bhgwge4w/crds/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /tmp/pip-pip-egg-info-e6sgjk2n
         cwd: /tmp/pip-install-bhgwge4w/crds/
    Complete output (17 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-bhgwge4w/crds/setup.py", line 79, in <module>
        import crds   #  local subdirectory...  ew...
      File "/tmp/pip-install-bhgwge4w/crds/crds/__init__.py", line 97, in <module>
        alias_subpackage_module("crds.refactoring", REFACTORING_MODULES)
      File "/tmp/pip-install-bhgwge4w/crds/crds/__init__.py", line 58, in alias_subpackage_module
        globals()[module] = importlib.import_module(subpkg + "." + module)
      File "/tmp/Initializationyciqwo9_/env/lib/python3.7/importlib/__init__.py", line 127, in import_module
        return _bootstrap._gcd_import(name[level:], package, level)
      File "/tmp/pip-install-bhgwge4w/crds/crds/refactoring/checksum.py", line 27, in <module>
        from crds.certify import mapping_parser
      File "/tmp/pip-install-bhgwge4w/crds/crds/certify/__init__.py", line 5, in <module>
        from .certify import *
      File "/tmp/pip-install-bhgwge4w/crds/crds/certify/certify.py", line 18, in <module>
        import numpy as np
    ModuleNotFoundError: No module named 'numpy'
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
```

**Comment by pllim** [comment]
@barentsen , do you think adding that notebook to https://github.com/spacetelescope/notebooks/blob/master/exclude_notebooks will workaround this problem until Drizzlepac team can fix it? Thanks! (I am not sure if `exclude_notebooks` is actually respected...)

**Comment by barentsen** [comment]
@pllim Adding the failing notebook to `exclude_notebooks` (cf. c3a357a) did not help.

The mechanism appears to be ignored by CircleCI, e.g. `grep exclude .circleci/*` does not return a match. The file is only used by the `convert.py` script in the root directory.

**Comment by pllim** [comment]
No worries. Thanks for checking!

**Comment by pllim** [comment]
p.s. @mustaric has asked me to help you get your PRs merged, so feel free to ping me as they come in. Good luck! 😄

**Comment by barentsen** [comment]
@pllim Thank you!! You already responded faster to this one than I could possibly have wished for!! 🙏

**Pull request merged**